pyproject.toml

```diff
@@ -110,3 +110,42 @@ minversion = "6.0"
 testpaths = "iris_grib"
 xfail_strict = "True"
 
+[tool.ruff]
+# Exclude the following, in addition to the standard set of exclusions.
+# https://docs.astral.sh/ruff/settings/#exclude
+line-length = 88
+src = [
+    "iris-grib",
+    "docs",
+]
+
+[tool.ruff.format]
```

Comment:
@HGWright We could enable formatting of docstrings with configuring the option [docstring-code-format](https://docs.astral.sh/ruff/settings/#format_docstring-code-format).

There may be new fall-out from enabling this.